eng/docs/api/assets/README.md

```diff
@@ -0,0 +1,255 @@
+```
+
+### header.html
+
+In the `<head>` of the page add the following snippets:
+
+```html
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-62780441-44"></script>
+<script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'UA-62780441-44');
+</script>
+```
+
+```html
+
+<script>
+WINDOW_CONTENTS = window.location.href.split("/");
+
+// language specific
+SELECTED_LANGUAGE = "cpp";
+populateOptions("#versionSelector", [
+  "#versionSelector",
+  "#versionSelectorHeader"
+]);
+
+function currentVersion() {
+  // Filled in by Doxygen template
+  return "$projectnumber";
+}
+
+function currentPackage() {
+  // Filled in by Doxygen template
+  return "$projectname";
+}
+
+function httpGetAsync(targetUrl, callback) {
+  var xmlHttp = new XMLHttpRequest();
+  xmlHttp.onreadystatechange = function() {
+    if (xmlHttp.readyState == 4 && xmlHttp.status == 200)
+      callback(xmlHttp.responseText);
+  };
+  xmlHttp.open("GET", targetUrl, true); // true for asynchronous
+  xmlHttp.send(null);
+}
+
+function showSelectors(selectors) {
+  selectors.forEach(function(item, index) {
+    $(item).show();
+  });
+}
+
+function hideSelectors(selectors) {
+  selectors.forEach(function(item, index) {
+    $(item).hide();
+  });
+}
+
+function populateOptions(optionSelector, otherSelectors) {
+  if (currentPackage()) {
+    var versionRequestUrl =
+      "https://azuresdkdocs.blob.core.windows.net/$web/" +
+      SELECTED_LANGUAGE +
+      "/" +
+      currentPackage() +
+      "/versioning/versions";
+
+    httpGetAsync(versionRequestUrl, function(responseText) {
+      if (responseText) {
+        let options = responseText.match(/[^\r\n]+/g);
+        populateVersionDropDown(optionSelector, options);
+        showSelectors(otherSelectors);
+
+        $(optionSelector).change(function() {
+          targetVersion = $(this).val();
+          window.location.href = getPackageUrl(SELECTED_LANGUAGE, currentPackage(), targetVersion);
+        });
+      }
+    });
+  }
+}
+
+function populateVersionDropDown(selector, values) {
+  var select = $(selector);
+
+  $("option", select).remove();
+
+  $.each(values, function(index, text) {
+    $("<option/>", { value: text, text: text }).appendTo(select);
+  });
+  select.val(currentVersion());
+}
+
+function getPackageUrl(language, package, version) {
+  return (
+    "https://azuresdkdocs.blob.core.windows.net/$web/" +
+    language +
+    "/" +
+    package +
+    "/" +
+    version +
+    "/index.html"
+  );
+}
+
+</script>
+```
+
+For the following look for and replace the sections bounded by
+`<!--BEGIN XXXX-->` and `<!--END XXXX-->` with the following:
+
+For the project logo section:
+
+```html
+  <!--BEGIN PROJECT_LOGO-->
+  <td id="projectlogo">
+    <img alt="Logo" src="$relpath^$projectlogo"/>
+  </td>
+  <!--END PROJECT_LOGO-->
+```
```

Comment:
Similarly here, can you add a couple sentences explaining why we want to do this modification? What's the net impact.

Reply:
This one might just be formatting. I grabbed these from the diff and stuck them directly in here.